loleaflet/js/global.js

```diff
@@ -3,7 +3,7 @@
 (function (global) {
 
 	global.getParameterByName = function (name) {
-		name = name.replace(/[\[]/, '\\[').replace(/[\]]/, '\\]');
```

Comment:
I'm not sure about this. Might need verification. What was the reason for this change?

Reply:
i checked it with regular expression validator, it is working. i think replace doesn't need to escape in range tokens for "[" character.

the eslint indicated the escape character is unnecessary. 

i couldn't find any document for this situation.